File: javascript/convertYesNo.js
// function boolToWord(bool) {
//   if bool === true {
//     return 'Yes'
//   } else if bool === false {
//     return 'No'
//   }
// }

//
// Test.assertEquals(boolToWord(true), 'Yes')
// Test.assertEquals(boolToWord(false), 'No')


const boolToWord = bool => bool ? console.log('Yes') : console.log('No');

boolToWord(true);


function boolToWord(bool) {
  if (bool) {
    return 'Yes';
  } else {
    return 'No';
  }
}

console.log(boolToWord(true));
//Yes

File: python/convertYesNo.py
def bool_to_word(boolean):
    if boolean == True:
        return 'Yes'
    else:
        return 'No'


print(bool_to_word(True))
# Yes





def bool_to_word(bool):
    return "Yes" if bool else "No"

print(bool_to_word(True))
# Yes






def bool_to_word(bool):
    if bool:
        return "Yes"
    return "No"

print(bool_to_word(bool))
# Yes
